plume-models/src/blogs.rs

```diff
@@ -142,8 +143,8 @@ impl Blog {
                     name: inst.clone(),
                     local: false,
                     // We don't really care about all the following for remote instances
-                    long_description: String::new(),
-                    short_description: String::new(),
+                    long_description: SafeString::new(&<String>::new()),
```

Comment:
you can change `&<String>::new` here to just `""`, since `SafeString::new()` takes a `&str`. It will save an allocation, at least, especially since currently the allocation just gets dropped and another is made in the `ammonia::clean` function.